### PR TITLE
pipeline updates

### PR DESCRIPTION
**Pull request by dmzoneill**

### CODE REVIEW

File: pr_check.sh [hunk 1]
```diff
@@ -1,44 +1,107 @@
...
+
+APP_NAME="tower-analytics"  # name of app-sre "application" folder this component lives in
+APP_NAME="$APP_NAME,gateway,insights-ephemeral"
+COMPONENT_NAME="tower-analytics-clowdapp"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+IMAGE="quay.io/cloudservices/automation-analytics-api"
+IMAGE_TAG="qa"
+
+IQE_PLUGINS="automation_analytics"
+IQE_MARKER_EXPRESSION="ephemeral"
 IQE_FILTER_EXPRESSION=""
```
Comment (jameswnl) [inline pr_check.sh:39]: remove Line37-39

**Review by jameswnl — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -109,7 +109,8 @@
     "start": "NODE_ENV=development webpack serve --config config/dev.webpack.config.js",
     "start:container": "NODE_ENV=development webpack serve --config config/dev.webpack.config.js --host 0.0.0.0",
     "build:prod": "NODE_ENV=production webpack --config config/prod.webpack.config.js",
-    "deploy": "npm-run-all build:prod",
+    "deploy": "npm-run-all build:prod lint test",
+    "deploy-only": "npm-run-all build:prod",
```
Comment (jameswnl) [inline package.json:113]: these changes are not needed any more. just use the `deploy` would do

File: .travis/pr_build.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
+  # if this is a PR build, backup TRAVIS_PULL_REQUEST and set it to false
+  # will be reset at end of custom_release.sh
+  TRAVIS_PULL_REQUEST_BAK="${TRAVIS_PULL_REQUEST}"
+  export TRAVIS_PULL_REQUEST_BAK
+  export TRAVIS_PULL_REQUEST=false
+  echo "Backup to TRAVIS_PULL_REQUEST_BAK: ${TRAVIS_PULL_REQUEST_BAK}"
+fi
+npm run deploy-only && curl -sSL https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master/src/bootstrap.sh | bash -s
```
Comment (jameswnl) [inline .travis/pr_build.sh:19]: change `deploy-only` to `deploy`

**Review by brumik — APPROVED** [review]
LGTM

**Pull request merged**